Skate_Behaviour_IO.m

```diff
@@ -73,6 +81,20 @@
         event_markers(strcmp(event_strings,'S  7')) = nFalseAlarm;   %false alarm
         event_markers(strcmp(event_strings,'S  9')) = nCorrectResponse;   %correct response
         
+        %if sum(strcmp(subs{i_sub}, {'124'; '125'; '126'; '127'; '128'; '129'}) > 0)
```

Comment:
change this to check for the dif_trig directly

Reply:
that part of the code is commented out. the part I use to search for those subjects directly is this one below: 

```
        if dif_trig(i_sub)
            nStandard = 1;
            nTarget = 2;
            nFalseAlarm = 3;
            nCorrectResponse = 4;
            
            event_markers = zeros(size(event_strings));
            event_markers(strcmp(event_strings,'S  1')) = pStandard;   %standard
            event_markers(strcmp(event_strings,'S  2')) = pTarget;   %target
            event_markers(strcmp(event_strings,'S  3')) = pFalseAlarm;   %false alarm
            event_markers(strcmp(event_strings,'S  4')) = pCorrectResponse;   %correct response
        end
```

Reply:
great, remove the commented out sections so the code is clear and easy for others to read

Reply:
Just did. Please see newer version.